### PR TITLE
[PD-2013] Fix Build for IE8

### PR DESCRIPTION
**Pull request by darrint**
This restores IE8 compatibility with our bundles.

I accomplished this by downgrading Babel from 6.x to 5.x. The Babel 6.x bug to watch:

https://phabricator.babeljs.io/T2817

See ticket for test guide.

### CODE REVIEW

File: gulp/src/nonuseroutreach/main.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import 'babel/polyfill';
```
Comment (darrint) [inline gulp/src/nonuseroutreach/main.js]: Eventually we will need this line in manageusers/main.

File: gulp/src/util/polyfills.js [hunk 2]
```diff
@@ -10,6 +10,11 @@ export function installPolyfills() {
     // This gives us the Promise API.
   es6PromisePolyfill();
 
+  // Needed on ie8 to make sure fetch can find Promise.
```
Comment (darrint) [inline gulp/src/util/polyfills.js]: For some reason I didn't need this with browserify/babel5 or webpack/babel6, but I do need it in webpack/babel5.

File: gulp/package.json [hunk 3]
```diff
@@ -9,16 +9,11 @@
...
-    "babel-eslint": "^5.0.0-beta4",
-    "babel-loader": "^6.2.0",
-    "babel-polyfill": "^6.2.0",
-    "babel-preset-es2015": "^6.2.4",
-    "babel-preset-react": "^6.2.4",
-    "babel-preset-stage-2": "^6.2.4",
-    "babel-register": "^6.2.0",
-    "babel-runtime": "^6.2.0",
+    "babel": "^5.8.23",
+    "babel-core": "^5.8.23",
```
Comment (darrint) [inline gulp/package.json]: The most time-consuming part of this fix was figuring out that I needed to pin the babel-core version at 5.x, as npm resolved it to 6.x.

File: gulp/src/nonuseroutreach/main.js (hunk 1 above)
Comment (aspidites) [inline gulp/src/nonuseroutreach/main.js]: If it's making IE 8 work, why not add it here?

Comment (darrint) [inline gulp/src/nonuseroutreach/main.js]: I don't want to create a merge conflict.

**Pull request merged**